lib/stormpath-sdk/resource/collection.rb

```diff
@@ -51,6 +51,10 @@ def limit limit
     self
   end
 
+  def size
+    current_page.items.size
```

Comment:
Hi guys - please use the Java SDK as the data model for collections.  It represents *exactly* what a Collection Resource should have across all of our SDKs:

https://github.com/stormpath/stormpath-sdk-java/blob/8e1449ee9e2db78664099faccb39e40b289bc3af/impl/src/main/java/com/stormpath/sdk/impl/resource/AbstractCollectionResource.java